templates/db.cfn.yml

```diff
@@ -122,22 +229,119 @@ Resources:
       StorageType: gp2
       MultiAZ: !If [ IsProd, true, false ]
       StorageEncrypted: !Ref EncryptionAtRest
+      MonitoringInterval: !If [ EnhancedMonitoringSupprtedAndEnabled, 60, 0 ]
+      MonitoringRoleArn: !If [ EnhancedMonitoringSupprtedAndEnabled, !GetAtt EnhancedMonitoringRole.Arn, !Ref "AWS::NoValue" ]
       CopyTagsToSnapshot: true
       Tags:
       - Key: Name
-        Value: !Ref "AWS::StackName"
+        Value: !Ref AWS::StackName
     DependsOn: DatabaseSubnetGroup
 
-  DatabaseSubnetGroup:
-    Type: AWS::RDS::DBSubnetGroup
+  DatabaseCpuAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Condition: AlarmsEnabled
```

Comment:
Is it possible that this condition could also be `EnhancedMonitoringSupportedAndEnabled`? i am unable to tell at a glance.